mongo/options/countoptions.go

```diff
@@ -15,6 +15,13 @@ type CountOptions struct {
 	// default value is nil, which means the default collation of the collection will be used.
 	Collation *Collation
 
+	// A string or document that will be included in server logs, profiling logs, and currentOp queries to help trace
+	// the operation.  The default is the empty interface, which means that no comment will be included in the logs.
+	//
+	// TODO(GODRIVER-2386): CountOptions executor uses aggregation under the hood, which means this type has to be
+	// TODO a string for now.  This can be replaced with `Comment interface{}` once 2386 is implemented.
+	Comment *string
```

Comment:
Keep in mind that the `TODO` here will be included with the generated docs for the [CountOptions](https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo/options#CountOptions) struct. I'm not sure if there's a better way to include that `TODO` to keep it out of the docs, so this seems fine. Do you know of any way to keep it out of the docs?

Reply:
We could use `// Bug(GODRIVER-2386)` in this case?  It will exclude those commetns according to the [docs](https://go.dev/blog/godoc):

>  Top-level comments that begin with the word "BUG(who)” are recognized as known bugs, and included in the “Bugs” section of the package documentation.

Reply:
That's a good point about the `// BUG` comments. However, those will still show up in the documentation for the struct field and I wouldn't really describe the behavior as a "bug". I tested a few comment styles and found that comments separated from fields are not included in the rendered documentation. Consider moving the `TODO` comments before the struct field comment to keep it out of the rendered documentation. Also consider updating "the empty interface" to "nil" in the field description.


E.g.
```go
// TODO(GODRIVER-2386): CountOptions executor uses aggregation under the hood, which means this type has to be
// TODO a string for now. This can be replaced with `Comment interface{}` once 2386 is implemented.

// A string or document that will be included in server logs, profiling logs, and currentOp queries to help trace
// the operation. The default is nil, which means that no comment will be included in the logs.
Comment *string
```